java/generate/generate.sh

```diff
@@ -68,7 +68,7 @@ rm -rf $sdk_output_folder/src/test/java/com/finbourne/lusid/model
 
 # update pom version
 sdk_version=$(cat $swagger_file | jq -r '.info.version')
-mvn -f $sdk_output_folder/pom.xml versions:set -DnewVersion=$sdk_version-SNAPSHOT
+mvn -f $sdk_output_folder/pom.xml versions:set -DnewVersion=$sdk_version
 rm -f $output_folder/.openapi-generator-ignore
```

Comment:
Do we need to add the gpg plug-in to the POM file for the package to be signed?

```
<build>
  <plugins>
    <plugin>
      <groupId>org.apache.maven.plugins</groupId>
      <artifactId>maven-gpg-plugin</artifactId>
      <version>1.5</version>
      <executions>
        <execution>
          <id>sign-artifacts</id>
          <phase>verify</phase>
          <goals>
            <goal>sign</goal>
          </goals>
        </execution>
      </executions>
    </plugin>
  </plugins>
</build>
```

Reply:
correct, when running the `deploy` task it will sign them

Reply:
good spot on the dependencies jar, will investigate

Reply:
Okay great. So on the POM.xml file do we need to add the `maven-gpg-plugin`? Seems like it wasn't necessary even though the docs say that you need it?

Reply:
looks like the dependencies jar got removed with this commit https://github.com/finbourne/lusid-sdk-java-preview/commit/3b20e97acae59a060b4960c3f5190e693e61725b#diff-880de2aacbceb2b255c9e42032e9046bfd764b36bb119e46054a7f76f3049722 in September and the current jars being published don't contain them.

Speaking to Regis, it is accepted practice to not publish the dependencies so given this change is already live, I would say its out of scope for the PR